Dockerfile

```diff
@@ -1,15 +1,18 @@
-FROM node:18-bullseye-slim
+FROM node:20-slim
 WORKDIR /app
+EXPOSE 9000
 
-RUN apt-get update
 RUN apt-get install -y git
-RUN rm -rf /var/lib/apt/lists/*
```

Comment:
why?

Reply:
also i would maybe combine these 3 commands into one `RUN`

Reply:
Never saw this in a Dockerfile before - why remove the lists?

Reply:
they aren't needed after the apt update/install routine is done, why keep them?
also i would say this is a pretty commonly used pattern when building an image, although im not sure if it actually does anything _here_ because its on a separate RUN step